tables.txt

```diff
@@ -1,18 +1,7 @@
- butter_bridge | Another article about Mitch                            |         13 | mitch | 2020-10-11 12:24:00 | https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700 |     0 |             0
- rogersop      | UNCOVERED: catspiracy to bring down democracy          |          5 | cats  | 2020-08-03 14:14:00 | https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700 |     0 |             2
- butter_bridge | Living in the shadow of a great man                    |          1 | mitch | 2020-07-09 21:11:00 | https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700 |   100 |            11
- butter_bridge | They're not exactly dogs, are they?                    |          9 | mitch | 2020-06-06 10:10:00 | https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700 |     0 |             2
- rogersop      | Seven inspirational thought leaders from Manchester UK |         10 | mitch | 2020-05-14 05:15:00 | https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700 |     0 |             0
- rogersop      | Student SUES Mitch!                                    |          4 | mitch | 2020-05-06 02:14:00 | https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700 |     0 |             0
- icellusedkars | Does Mitch predate civilisation?                       |          8 | mitch | 2020-04-17 02:08:00 | https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700 |     0 |             0
- icellusedkars | Am I a cat?                                            |         11 | mitch | 2020-01-15 22:21:00 | https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700 |     0 |             0
- icellusedkars | Z                                                      |          7 | mitch | 2020-01-07 14:08:00 | https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700 |     0 |             0
-(13 rows)
+ comment_id |                                                     body                                                     | article_id |    author     | votes |     created_at      
+------------+--------------------------------------------------------------------------------------------------------------+------------+---------------+-------+---------------------
+         14 | What do you see? I have no idea where this will lead us. This place I speak of, is known as the Black Lodge. |          5 | icellusedkars |    16 | 2020-06-09 06:00:00
+         15 | I am 100% sure that we're not completely sure.                                                               |          5 | butter_bridge |     1 | 2020-11-24 00:08:00
+(2 rows)
```

Comment:
Good idea to visualise this, but we don't need to see it, consider git ignoring this file so its just for you. 🙂